carbon-design-system/carbon
# fix(type): reinstate helper-text-02

[Surfaced on slack](https://ibm-studios.slack.com/archives/C0M053VPT/p1659736231575189)

I'm unsure why this was deleted in https://github.com/carbon-design-system/carbon/pull/11078

#### Changelog

**Changed**

- reinstated `helper-text-02`

#### Testing / Reviewing

* Check to make sure `helper-text-02` is included in the storybook and available in the build
* Make sure I didn't miss any other spots that need to be updated. I basically made sure anywhere `helper-text-01` is present, `helper-text-02` is present alongside it.
